subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/AllDistributionIntegrationSpec.groovy

```diff
@@ -36,7 +36,7 @@ class AllDistributionIntegrationSpec extends DistributionIntegrationSpec {
 
     @Override
     int getMaxDistributionSizeBytes() {
-        return 212 * 1024 * 1024
+        return 215 * 1024 * 1024
```

Comment:
💭 These tests were broken, but for some reason never ran on `release`. @blindpirate is investigating.

Reply:
Reverted this. The problem was caused by the long branch name being included in the zip over and over.